packages/sm-commons/methods/actions.js

```diff
@@ -39,39 +38,6 @@ function readConfig(p) {
   }
 }
 
-function readJsonPackage(p) {
```

Comment:
is this function not used any-where?

Reply:
no, that was an old start command apparently. It's coming from the cherry pick.